src/Altinn.Platform/Altinn.Platform.Common/Common/Common/Model/Actor.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Common.Model
```

Comment:
Models for register and profile are placed in Altinn.Core Servicelibrary. We should probably co-locate all models related to the bridge so that maintenance is easier. Either move this class to servicelibrary or move the remaning classes to this namespace.

Reply:
Ya, It needs to be discussed. We will take a short discussion today and agree on where the common models for platform and studio must be placed